extendr-api/src/na.rs

```diff
@@ -23,9 +23,11 @@ lazy_static! {
     };
 }
 
-/// Return true if this primitive is NA.
+/// Trait signifying if a given type may take the value `NA` in the R sense.
```

Comment:
This isn't clear. Does this trait mean it can be an NA scalar, or it can be a vector that contains NA?

Reply:
I'm still not satisfied here. "take" is overloaded, and "in the R sense" is not precise enough.

Reply:
What about 

> Trait signifying if a given type may be represented as an `NA` value in R